compliance/collection/inventory/convert.go

```diff
@@ -0,0 +1,100 @@
+package inventory
+
+import (
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/uuid"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+)
+
+// ToNodeInventory converts a NodeInventory response to a native storage.NodeInventory version
+func ToNodeInventory(resp *scannerV1.GetNodeInventoryResponse) *storage.NodeInventory {
+	return &storage.NodeInventory{
+		NodeId:     uuid.Nil.String(),
```

Comment:
I assume this is because `NodeId` will be filled only in sensor, with k8s data, right? If that's the case, do you mind adding a single line comment here saying that we add `Nil` uuid as a placeholder and the real ID will be set in sensor?

Reply:
That is correct. The `NodeId` is not available to Compliance or the new NodeScanner.
Sensor is the first component in the message chain that can correlate a `NodeName` to a `NodeID`.
I've added a comment that this is a placeholder, thanks!